ludwig/data/preprocessing.py

```diff
@@ -1530,10 +1553,29 @@ def precompute_fill_value(dataset_cols, feature, preprocessing_parameters: Prepr
 @DeveloperAPI
 def handle_missing_values(dataset_cols, feature, preprocessing_parameters: PreprocessingConfigDict, backend):
     missing_value_strategy = preprocessing_parameters["missing_value_strategy"]
-
-    # Check for the precomputed fill value in the metadata
     computed_fill_value = preprocessing_parameters.get("computed_fill_value")
+    _handle_missing_values(dataset_cols, feature, missing_value_strategy, computed_fill_value, backend)
 
+
+@DeveloperAPI
+def handle_outliers(dataset_cols, feature, preprocessing_parameters: PreprocessingConfigDict, metadata, backend):
+    outlier_strategy = preprocessing_parameters.get("outlier_strategy")
+    if outlier_strategy is None:
+        return
+
+    outlier_threshold = preprocessing_parameters["outlier_threshold"]
+    computed_fill_value = preprocessing_parameters.get("computed_outlier_fill_value")
+
+    # Identify all outliers and set them to NA so they can be removed
+    s = dataset_cols[feature[COLUMN]]
```

Comment:
nit: can we use a better variable name

Reply:
Also, for observability reasons, can we add the number of rows in the column we found/replaced outliters for?

Reply:
`s` is for "series", yes. `column` usually refers to a column name in Pandas terminology. Happy to change it to `series` though.

Reply:
Problem with printing the number of rows removed is that, because of Dask's lazy evaluation, it will be very slow, since we'll need to recompute some things twice (once to print the count, once to apply the replacement). So I would prefer not to, which is consistent without how we handle missing values at the moment.

Reply:
That's completely fair - I do think we should figure out a mechanism for better observability - maybe using the verbosity level flag we pass around in Ludwig. Some users may be willing to wait longer for their preprocessing/training so that they can see these kinds of things that happen internally within Ludwig. 

Perfectly ok to skip for now!